Assets/Scripts/Layers/Properties/PropertyPanels/TransformPropertySection.cs

```diff
@@ -122,57 +133,127 @@ private void SetTransformLocks()
 
         private void OnPositionChanged(string axisValue)
         {
-            double.TryParse(position.xField.Text, out var x);
-            double.TryParse(position.yField.Text, out var y);
-            double.TryParse(position.zField.Text, out var z);
+            var numberFormat = new NumberFormatInfo
+            {
+                NumberDecimalSeparator = "."
```

Comment:
strings die vaker gebruikt gaan worden altijd cachen. dit is veel memory allocatie, dus bv. NumberDecimalSeparator = defaultNumberSeperator;